.github/workflows/integration_tests.yml

```diff
@@ -396,34 +408,15 @@ jobs:
       - name: Expose Deployed Services via Istio (for CD use-case)
         if: env.RUN_CONTINUOUS_DELIVERY_TEST == 'true'
         run: |
-          # install public-gateway.istio-system
-          kubectl apply -f - <<EOF
-          apiVersion: networking.istio.io/v1alpha3
-          kind: Gateway
-          metadata:
-            name: public-gateway
-            namespace: istio-system
-          spec:
-            selector:
-              istio: ingressgateway # use Istio default gateway implementation
-            servers:
-            - port:
-                number: 80
-                name: http
-                protocol: HTTP
-              hosts:
-              - "*"
-          EOF
```

Comment:
This is not needed anymore since it's namespace/testrun independent and therefore statically deployed on the both integration test clusters